config/v1/types_cluster_operator.go

```diff
@@ -149,7 +149,10 @@ const (
 	// Progressing indicates that the operator is actively rolling out new code,
 	// propagating config changes, or otherwise moving from one steady state to
 	// another.  Operators should not report progressing when they are reconciling
-	// a previously known state.
+	// (without action) a previously known state.  If the observed cluster state
+	// has changed and the operator/operand is reacting to it (scaling up for instance),
```

Comment:
nit: seems simpler to me to include "scaling" in the list of example changes above^, like:

> Progressing indicates that the operator is actively rolling out new code, propagating config changes, scaling pods, or otherwise moving from one steady state to another...

Reply:
This is a different sort of allowable change.  Rather than the strict enumeration above, this is an explicit indication that progressing=true when reacting to changes in reality.  It is clarifying the previous sentence about not going progressing under certain cases.